src/FixedPointDecimals.jl

```diff
@@ -36,9 +36,22 @@ export checked_abs, checked_add, checked_cld, checked_div, checked_fld,
 
 using Base: decompose, BitInteger
 
-import BitIntegers  # For 128-bit _widemul / _widen
+using BitIntegers: BitIntegers, UInt256, Int256
```

Comment:
Why not import the other used types?
```suggestion
using BitIntegers: BitIntegers, Int256, Int512, UInt256, UInt512
```

Reply:
Seems reasonable. The (U)Int256 were used much more heavily than (U)Int512. So I was inclined to refer to them directly through the qualified name. But I think your suggestion is better.